src/main/java/seedu/trackermon/logic/parser/CliSyntax.java

```diff
@@ -10,11 +10,11 @@
     public static final Prefix PREFIX_STATUS = new Prefix("s/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_COMMENT = new Prefix("c/");
-    public static final Prefix PREFIX_SORT_NAME_ACS = new Prefix("sna/");
-    public static final Prefix PREFIX_SORT_STATUS_ACS = new Prefix("ssa/");
-    public static final Prefix PREFIX_SORT_NAME_DES = new Prefix("snd/");
-    public static final Prefix PREFIX_SORT_STATUS_DES = new Prefix("ssd/");
-    public static final Prefix PREFIX_SORT_STATUS_ORD = new Prefix("so/");
+    public static final Prefix PREFIX_SORT_NAME = new Prefix("sn/");
+    public static final Prefix PREFIX_SORT_STATUS = new Prefix("ss/");
+    public static final Prefix PREFIX_SORT_RATING = new Prefix("sr/");
+    public static final Prefix PREFIX_SORT_TAGS = new Prefix("st/");
+    public static final Prefix PREFIX_SORT_ORDER = new Prefix("so/");
```

Comment:
Could you just have used the prefix we already have? `n/ s/ t/ r/`?

Reply:
ok